### DISCUSSION TITLE
[IDEA] Mail Server

### DISCUSSION THREAD

**Discussion by ljfreelancer88** [roadrunner-server/roadrunner#766] [Ideas]
I have checked spiral.dev and roadrunner.dev but
can't found on the documentation if it is possible to create a mail server that suitable for larger volumes of email in a loop. Adding this as a feature or add in the documentation would be helpful and appreciated.

**Comment by rustatian** [comment]
Hey @ljfreelancer88 . No, it's not possible at the moment, because the mail server works using SMTP protocol which is not supported at the moment. Let's discuss here, what do you want from this server, how do you think it should work, etc.

**Reply by rustatian** [reply]
@ljfreelancer88 Starting RR 2.6.0 we will show a new plugin to handle raw TCP payloads. So, you'll be able to create a worker to handle SMTP or any other protocol with the RR. Here is the client library: https://github.com/spiral/roadrunner-tcp.
You may also have a look at this project https://github.com/buggregator/app which is used the new `tcp` plugin to handle SMTP packets.
